jena/TemporalIndexProcessor/src/com/bbn/parliament/jena/graph/index/temporal/extent/TemporalInterval.java

```diff
@@ -175,6 +183,33 @@ public boolean finishedBy(TemporalInterval interval) {
 			&& interval.getStart().greaterThan(getStart());
 	}
 
+	@Override
+	public boolean equals(Object other) {
+		boolean areEqual = this == other;
+		if (!areEqual && TemporalInterval.class.equals(other.getClass())) {
+			TemporalInterval that = (TemporalInterval) other;
+			areEqual = this.start.instant == that.start.instant && this.end.instant == that.end.instant;
```

Comment:
I note that you've used == rather than equals().  Does this deserve further discussion, or are we pretty sure that we just need to not push our luck with too many changes, here?

Reply:
Nevermind -- stupid question:  `instant` is a long!!!